app/tweet_parser.py

```diff
@@ -26,7 +26,7 @@ def parse_timeline_status(status):
         "status_text": parse_string(parse_full_text(status)),
         "created_at": generate_timestamp(status.created_at),
 
-        "geo": status.geo,
+        "geo": str(status.geo), # think this is a nested object
```

Comment:
Yes, we are seeing objects like this: `{'type': 'Point', 'coordinates': [7.8340857, 98.2965371]}`